lib/adzerk/flight.rb

```diff
@@ -1,12 +1,20 @@
 module Adzerk
   class Flight < ApiEndpoint
     def countries
-      parse_response(@client.get_request('countries'))
```

Comment:
What was the issue with using `parse_response`?

Reply:
Updated